### ISSUE TITLE
How to install latest version of Talos in Google Colab?

### ISSUE THREAD

**Issue by SamBelkacem** [autonomio/talos#471]
I'm trying to install the latest version of Talos in **Google Colab**.

I used pip as follows: `pip install talos==0.6.6`

However, I noticed that I get the version 0.6.4 instead when I print `ta.__version__` 

How can I fix that?

Thanks

**Comment by mikkokotila** [comment]
Interesting. Can you try `pip install talos -U` or then you could also do `pip install git+https://github.com/autonomio/talos` and that should get you the latest version.

**Comment by SamBelkacem** [comment]
> Interesting. Can you try `pip install talos -U` or then you could also do `pip install git+https://github.com/autonomio/talos` and that should get you the latest version.

I tried both of them and I still get the version 0.6.4 when I print `ta.__version__`

What's really weird is when I try to reinstall Talos using  `pip uninstall talos` it displays `Uninstalling talos-0.6.7`.

How is that possible? Is something wrong with `ta.__version__`?

**Comment by mikkokotila** [comment]
And this is happening in a totally remote Google colab notebook that does not have any connection with your local machine or some remote host?

**Comment by SamBelkacem** [comment]
Yes, it has nothing to do with my machine. The code is run on the cloud of Google Colab.

I finally managed to install the latest version of Talos using `pip install git+https://github.com/autonomio/talos`

I still get version 0.6.4 when I print `ta.__version__` but I don't get any error in my code after the update. I believe Talos was updated to version 0.6.7 anyway. 

There is probably something wrong with `ta.__version__` which does not display the currently installed version.
